tox.embedapi.ini

```diff
@@ -17,9 +17,8 @@ install_command =
     # Install requirements in multiple steps because we don't want to install
     # Sphinx from `requirements/pip.txt` but from the `deps=` field.
     sh -c ' \
-        cat {toxinidir}/requirements/pip.txt | grep -v "Sphinx" > {toxinidir}/requirements/embedapi.txt; \
-        sed {toxinidir}/requirements/testing.txt -e "s|pip.txt|embedapi.txt|g" > {toxinidir}/requirements/testing.embedapi.txt; \
-        pip install -r {toxinidir}/requirements/testing.embedapi.txt; \
+        cat {toxinidir}/requirements/testing.txt | grep -v "Sphinx" > {toxinidir}/requirements/embedapi.txt; \
+        pip install -r {toxinidir}/requirements/embedapi.txt; \
```

Comment:
we were installing the Sphinx version from our requirements... guess this was done before we migrated to pip-tools.